src/amo/components/HomeHero/index.js

```diff
@@ -0,0 +1,93 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import Link from 'amo/components/Link';
+import translate from 'core/i18n/translate';
+import Hero from 'ui/components/Hero';
+import type { I18nType } from 'core/types/i18n';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+type InternalProps = {|
+  i18n: I18nType,
+|};
+
+export class HomeHeroBase extends React.Component<InternalProps> {
```

Comment:
How about we call this component `HomeHeroGuides`?

Reply:
You can probably use `React.PureComponent` here, see: https://reactjs.org/docs/react-api.html#reactpurecomponent

Reply:
oh? okay, I'll try that..